Update interfaces/web-nfc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/1d1a9ed/whatwg/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/467371826

lukebjerring reviewed (changes requested):
Will need to add the source of `EventTarget` to the deps in the test. Pretty sure that's `dom`.

autofoolip committed:
Update interfaces/web-nfc.idl

Source: https://github.com/tidoust/reffy-reports/blob/1d1a9ed/whatwg/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/467371826
